src/Serialization/Rings.jl

```diff
@@ -524,28 +498,24 @@ function load_object(s::DeserializerState,
 end
 
 # elements
-@registerSerializationType(Generic.LaurentSeriesFieldElem, "LaurentSeriesFieldElem")
-@registerSerializationType(Generic.LaurentSeriesRingElem, "LaurentSeriesRingElem")
-@registerSerializationType(ZZLaurentSeriesRingElem)
-type_needs_params(::Type{<: Union{ZZLaurentSeriesRingElem,
-                                  Generic.LaurentSeriesFieldElem,
-                                  Generic.LaurentSeriesRingElem}}) = true
+@register_serialization_type Generic.LaurentSeriesFieldElem "LaurentSeriesFieldElem" uses_params
+@register_serialization_type Generic.LaurentSeriesRingElem "LaurentSeriesRingElem" uses_params
+@register_serialization_type ZZLaurentSeriesRingElem uses_params
```

Comment:
Meta-question (not a blocker for this PR): I wonder, do we really want to serialize `ZZLaurentSeriesRingElem` under that name? Shouldn't a "laurent ring over the integers" automatically be mapped to `ZZLaurentSeriesRingElem`?

I am thinking here of how this might look longterm, in a FAIR format -- the existence of a separate `ZZLaurentSeriesRingElem` type seems like an OSCAR specific implementation detail to me...